src/bootstrap/setup.rs

```diff
@@ -50,6 +51,16 @@ impl Profile {
         }
         out
     }
+
+    pub fn as_str(&self) -> &'static str {
```

Comment:
Can you change `impl Display for Profile` to use this function, so they can't get out of sync?

Reply:
implemented, please check!